controllers/v1/events/event.controller.js

```diff
@@ -327,6 +320,32 @@ const remove = async (request, response) => {
   return response.status(responseCode).json(responseData)
 }
 
+const mongoEventData = (newEvent) => {
+  let eventData = {
+    name: newEvent.name,
+    date: newEvent.date,
+    headquarter: newEvent.headquarter,
+  }
+  eventData['address'] = newEvent.address ? newEvent.address : ''
+  return eventData
+}
+
+const firebaseEventData = (newEvent) => {
```

Comment:
Podemos usar destructuring aca
```suggestion
const firebaseEventData = ({name, date, headquarter, responsable }) => {
```